CodeSystem/CodeSystem-DM-prescription-type.xml

```diff
@@ -2,94 +2,97 @@
 <CodeSystem xmlns="http://hl7.org/fhir">
    <id value="prescription-type" />
    <url value="https://fhir.nhs.uk/CodeSystem/prescription-type" />
-   <version value="1.1.6" />
+   <version value="2.0.0" />
```

Comment:
Is this a major version update? I'd say the purpose of the asset is the same, just with some additions, so probably a minor uplift